misk-slack/src/main/kotlin/misk/slack/SlackModule.kt

```diff
@@ -33,4 +36,9 @@ class SlackModule(private val config: SlackConfig) : KAbstractModule() {
         .build()
     return retrofit.create(SlackWebhookApi::class.java)
   }
+  @Provides @com.google.inject.Singleton @Named("emailBridge") fun provideMoshi(): Moshi? {
```

Comment:
Moshi? –> Moshi

Why fully-qualify `@Singleton` ?

Reply:
emailBridge ?!

Reply:
🤦‍♂ ...
Better?
Am I missing anything that you know of?